spinbot/event/pull_request_closed_event_handler.py

```diff
@@ -0,0 +1,43 @@
+import re
+from datetime import datetime
+from .pull_request_event import GetPullRequest
+from .handler import Handler
+
+class PullRequestClosedEventHandler(Handler):
+    def __init__(self):
+        super().__init__()
+
+    def handles(self, event):
+        return (event.type == 'PullRequestEvent'
+            and event.payload.get('action') == 'closed')
+
+    def handle(self, g, event):
+        pull_request = GetPullRequest(g, event)
+        if not pull_request:
+            log.warn('Unable to determine pull request for {}'.format(event))
+            return
+
+        merged = pull_request.merged
+        age = pull_request.closed_at - pull_request.created_at
+        change_size = pull_request.additions + pull_request.deletions
+        files_changed_count = pull_request.changed_files
+        comments = pull_request.comments
+
+        approved = next((r for r in pull_request.get_reviews() if r.state == 'APPROVED'), None) is not None
```

Comment:
Do we care about self-approvals here? I hope it wouldn't happen, but you never know.

Reply:
They're no longer possible via github

Reply:
![image](https://user-images.githubusercontent.com/4874941/40324136-9ea99a44-5d05-11e8-8f86-981341aa08da.png)